### PR TITLE
feat: add optional `dataCy` attribute

### PR DESCRIPTION
**Pull request by mstruebing**
This adds an optional `dataCy` attributes
to `ConfigurationMap{Label,Row,Value}` to be able
to search efficiently with cypress for that values

Related to D2IQ-81001

## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [ ] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [x] Info for applicable sections above is provided

### CODE REVIEW

**Review by clintonmedbery — APPROVED** [review]
👍 Nice!

**Review by vacas5 — APPROVED** [review]

File: packages/configurationmap/components/ConfigurationMapValue.tsx [hunk 1]
```diff
@@ -3,10 +3,18 @@ import { cx } from "@emotion/css";
 import { flexItem, breakWord } from "../../shared/styles/styleUtils";
 import { ddReset } from "../../shared/styles/styleUtils/resets/definitionListReset";
 
-const ConfigurationMapValue = ({ children }) => (
+interface ConfigurationMapValueProps {
+  children: React.ReactNode;
+  dataCy?: string;
+}
+
+const ConfigurationMapValue: React.FC<ConfigurationMapValueProps> = ({
```
Comment (vacas5) [inline packages/configurationmap/components/ConfigurationMapValue.tsx:11]: we've been deprecating use of React.FC

**Review by vacas5 — CHANGES_REQUESTED** [review]

File: packages/configurationmap/components/ConfigurationMapValue.tsx (hunk 1 above)
Comment (mstruebing) [reply packages/configurationmap/components/ConfigurationMapValue.tsx]: updated.

**Review by vacas5 — APPROVED** [review]

**Review by nataliepina — APPROVED** [review]
Thanks for making that update :)

**Pull request merged**